ansible/roles_ocp_workloads/ocp4_workload_performance_monitoring/defaults/main.yml

```diff
@@ -0,0 +1,22 @@
+# generic
+become_override: false
+verify_tls: false
+insecure_skip_tls_verify: true
+silent: false
+
+# workshop variables
```

Comment:
we suggest adding the role name to all of the "public" variables to avoid collisions.

For example: ocp4_workload_performance_monitoring_workshop_user_role